packages/@aws-cdk/aws-sagemaker-alpha/lib/instance-type.ts

```diff
@@ -204,6 +204,46 @@ export class InstanceType {
    */
   public static readonly G5_XLARGE = InstanceType.of('ml.g5.xlarge');
 
+  /**
+   * ml.g5.12xlarge
```

Comment:
It should be `ml.g6.12xlarge`. Same to all new instances.

Reply:
good catch, rebased and updated